lib/ansible/modules/cloud/azure/azure_rm_acs.py

```diff
@@ -42,6 +42,7 @@
     orchestration_platform:
         description:
             - Specifies the Container Orchestration Platform to use. Currently can be either DCOS, Kubernetes or Swarm.
+            - If specifies the Container Orchestration Platform with Kubernetes, need specify the service_principal
```

Comment:
`I(service_principal) must be  defined if set to C(Kubernetes).`
Also add one line to service_principal description `Required when I(orchestration_platform=Kubernetes).`

The symbols definition you can find at https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation . Please also check these symbols in the documentation. I believe there are many wrong use.

Reply:
Thanks yuwzho. Has changed according to comments!